README.md

```diff
@@ -94,12 +94,14 @@ const newman = require('newman'); // require newman in your project
 newman.run({
     collection: require('./sample-collection.json'),
     reporters: 'cli'
-}, function (err) {
+}, function (err, data) {
 	if (err) { throw err; }
-    console.log('collection run complete!');
+    console.log('collection run complete! Summary Run Data', data);
 });
 ```
 
+The collection may either be a URL or a path to a local JSON file.  If it is a URL, you must provide the `postmanApiKey` property.
```

Comment:
Well, you can give a path to the collection file or any URL where the collection JSON is hosted.
`postmanApiKey` is only needed if fetched via Postman API.